test/message/test_runner_desctibe_it.out

```diff
@@ -101,16 +99,24 @@ not ok 11 - async throw fail
     *
     *
   ...
+# Subtest: async skip fail
+not ok 12 - async skip fail
+  ---
+  duration_ms: *
+  failureType: 'callbackAndPromisePresent'
+  error: 'passed a callback but also returned a Promise'
```

Comment:
That might be out of scope for the PR: should this really end up with this error message?
I would hope for e.g., `"Test 'Foobar' is meant to be skipped but code that ran after calling .skip() threw an error."`